ctxtimecheck/ctxtimecheck.go

```diff
@@ -21,18 +22,17 @@ var Analyzer = &analysis.Analyzer{
 }
 
 func run(pass *analysis.Pass) (any, error) {
-	in := pass.ResultOf[ssainspect.Analyzer].(*ssainspect.Inspector)
+	seq := pass.ResultOf[ssainspect.Analyzer].(iter.Seq[*ssainspect.Cursor])
```

Comment:
ssainspect has a bug in v0.2.0.
I fixed the bug in following PR.
https://github.com/gostaticanalysis/ssainspect/pull/8